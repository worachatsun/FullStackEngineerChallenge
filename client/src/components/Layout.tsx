import React, { FunctionComponent } from 'react';
import Header from './commons/Header';
import RouterCompoment from './Router';

const Layout: FunctionComponent = ({ children }) => {
  return (
    <>
      {/* <Header /> */}
      <RouterCompoment />
    </>
  );
};

export default Layout;
