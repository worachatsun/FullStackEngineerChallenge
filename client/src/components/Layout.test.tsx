import React from "react";
import renderer from "react-test-renderer";
import Layout from "./Layout";

describe("<Layout />", () => {
    test("should render correctly", () => {
        const render = renderer.create(<Layout />).toJSON();
        expect(render).toMatchSnapshot();
    });
});
