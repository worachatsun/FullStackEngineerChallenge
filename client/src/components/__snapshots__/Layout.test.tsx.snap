// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> should render correctly 1`] = `
Array [
  <div
    className="sc-AxirZ fqNgBu"
  >
    <div
      className="sc-AxiKw jJjpKK"
      onClick={[Function]}
    >
      <img
        alt="logo"
        src="img_logo.png"
      />
    </div>
    <div
      className="sc-AxhCb cupTmd"
    >
      <div
        className="sc-AxhUy iYMzkw"
      >
        
      </div>
      <button
        className="sc-AxjAm fbqdBC"
        onClick={[Function]}
      >
        Logout
      </button>
    </div>
  </div>,
  <div
    className="sc-AxheI jdaMWS"
  >
    <div
      className="sc-AxgMl kikqEL"
    />
  </div>,
]
`;
